### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'text') when an error does not have the file prop

### ISSUE THREAD

**Issue by btakita** [ai/check-dts#30]
https://github.com/ai/check-dts/blob/main/check.js#L114

If the `error` does not have a `file` prop, the error occurs:

```
⠧ Check typesTypeError: Cannot read properties of undefined (reading 'text')
    at check (file:///home/brian/work/ctx-core/ctx-core/node_modules/.pnpm/check-dts@0.6.7_oofdlok2qnlrhnlln2uverlm2m/node_modules/check-dts/check.js:114:48)
    at async run (file:///home/brian/work/ctx-core/ctx-core/node_modules/.pnpm/check-dts@0.6.7_oofdlok2qnlrhnlln2uverlm2m/node_modules/check-dts/bin.js:24:18)
```

**Comment by ai** [comment]
Why error doesn't have file?

**Comment by btakita** [comment]
The error was caused by an invalid `tsconfig.json`.

Here is the error object:

```js
{
  messageText: "Option 'declarationDir' cannot be specified without specifying option 'declaration' or option 'composite'.",
  fileName: undefined,
  start: undefined,
  code: 5069
}
```

**Comment by ai** [comment]
We can `thrown` this type of errors out of app to crash CLI.

Is it OK? Do you want to send PR (it should be simple and will save your name in the project’s history)?

**Comment by ai** [comment]
@myshov sure!

**Comment by ai** [comment]
The fix was released in 0.7.2.
